lib/matplotlib/tests/test_dates.py

```diff
@@ -457,3 +457,27 @@ def test_DayLocator():
 def test_tz_utc():
     dt = datetime.datetime(1970, 1, 1, tzinfo=mdates.UTC)
     dt.tzname()
+
+
+@pytest.mark.parametrize("x, tdelta",
+                         [(1, datetime.timedelta(days=1)),
+                          ([1, 1.5], [datetime.timedelta(days=1),
+                                      datetime.timedelta(days=1.5)])])
+def test_num2timedelta(x, tdelta):
+    dt = mdates.num2timedelta(x)
+    assert dt == tdelta
+
+
+@pytest.mark.parametrize('dt, fmt, out',
```

Comment:
I think you'd probably want a lot more tests. What happens for a negative `timedelta`?

How about `('{H:02}:{M:02}', timedelta(days=4, hours=0, minutes=0, seconds=4))`?